lib/ingreedy/continuous_language_locale.rb

```diff
@@ -0,0 +1,9 @@
+module Ingreedy
+  module ContinuousLanguageLocale
+    CONTINUOUS_LANGUAGES_LOCALES = [:ja].freeze
```

Comment:
I think we can.
For Japanene, the JP development team had reached an agreement, but if it looks like it could be added in China and Thailand, I will add them.